Work around a deficiency in cmake's build rule generation.
Upstream: not yet

--- libphonenumber-8.12.20.orig/cpp/CMakeLists.txt
+++ libphonenumber-8.12.20/cpp/CMakeLists.txt
@@ -351,7 +351,7 @@
   ${TEST_METADATA_TARGET}
   "${RESOURCES_DIR}/PhoneNumberMetadataForTesting.xml"
   "test_metadata"
-  "metadata"
+  "test_metadata"
 )
 list (APPEND TESTING_LIBRARY_SOURCES "src/phonenumbers/test_metadata.cc")
 
